.ci/setenvconfig

```diff
@@ -201,17 +201,17 @@ CFG
 
 clusterName=$2
 stackVersion=$3
+snapshotVersion=$(case $stackVersion in (*-SNAPSHOT)  echo Yes;; esac)
```

Comment:
This is a kind of boolean _(even this could be misleading since if we only check if it is set)_, maybe rename it to `isSnapshotVersion=...` ?